puma/tests/utils/test_histogram_utils.py

```diff
@@ -241,6 +241,16 @@ def setUp(self):
             0.05,
             0.025,
         ])
+        self.step_rsd = np.array([4, 4, -5.1961524, 0, -4.8989795, -8, -11.8321596])
+        self.step_rsd_unc = np.array([
+            0.625,
+            0.625,
+            0.5773503,
+            0,  # for RSD, if num=denom error, error is 0? or num_unc
```

Comment:
Question here more about the logic of uncertainty calculations. If there is an error associated with the numerator, but `num=denom`, should the error be zero or equal to `num_unc`?

Reply:
Good q, we had an issue about this https://github.com/umami-hep/puma/issues/213, which may have been resolved prematurely. I think if we are calculating the error on the ratio, we don't need to put an error on the nominal line at y=1 on the ratio panel.

Reply:
I reopened https://github.com/umami-hep/puma/issues/213, we can use that to follow up on this, I'm happy to merge this now

Reply:
Give me until he end of today please, I will double check with some JetEtMiss people about the RSD, whether we need to show the numerator uncertainty for the reference line (y=0). I guess for consistency with the ratio we shouldn't here either, but I wanted to check

Reply:
UPD: yeah 0 is fine for RSD @samvanstroud, from my side I think this is good to merge